### PR TITLE
My calendar service integration

### PR DESCRIPTION
**Pull request by AlexJYoungman**

### CODE REVIEW

**Comment by AlexJYoungman** [comment]
[HOLD] Do not merge until web services are available

**Comment by AlexJYoungman** [comment]
The MyCalendar page has been set up to use the web service when it becomes available, it shouldn't require much tweaking at all when the services are ready.

File: app/components/CreateAbsence.js [hunk 1]
```diff
@@ -64,13 +66,52 @@ var CreateAbsence = React.createClass({
...
+
+    var returnObject = {
+      start_date: this.applyHours(fromDate),
+      end_date: this.applyHours(toDate),
+      type: type,
+      total_days: this.countDays(fromDate, toDate),
+      status: 'PENDING'
+    }
+
+    console.log(JSON.stringify(returnObject));
```
Comment (joerushbrooke) [inline app/components/CreateAbsence.js]: Remove please

File: app/components/CreateAbsence.js [hunk 2]
```diff
@@ -64,13 +66,52 @@ var CreateAbsence = React.createClass({
...
 
-    this.props.createAbsence(fromDate, toDate, reason);
+    // this.props.createAbsence(fromDate, toDate, reason);
```
Comment (joerushbrooke) [inline app/components/CreateAbsence.js]: Unneccessary comment

**Pull request merged**